worker/raft/raftleaseservice/manifold.go

```diff
@@ -112,7 +113,7 @@ func (config ManifoldConfig) start(context dependency.Context) (worker.Worker, e
 		return nil, errors.Trace(err)
 	}
 
-	var r RaftApplier
+	var r *raft.Raft
```

Comment:
Even though we only use `RaftApplier` trying to get just that from the `context.Get` causes all sorts of issues.